docs/using/libraries/java/index.rst

```diff
@@ -212,18 +217,18 @@ Likewise, dictionaries constructed in Polar may be passed into Java methods.
 
 Enumerations
 ^^^^^^^^^^^^
-Oso handles Java objects that implement the `Enumeration <https://docs.oracle.com/javase/10/docs/api/java/util/Enumeration.html>`_ interface by evaluating each of the
-object's elements one at a time:
+You may iterate over a Java `Enumeration <https://docs.oracle.com/javase/10/docs/api/java/util/Enumeration.html>`_
```

Comment:
Not sure how the Java bits of #485 are implemented, but (assuming they're the same as the Ruby bits I looked at) can't we now enumerate over anything we can convert into an enumeration? I.e., the below `return Collections.enumeration(List.of("HR", "payroll"));` could be `return List.of("HR", "payroll");`?

Reply:
There are several cases: collections, iterables, enumerations. As far as I can tell, the others reduce to the last, so I think it's fine.

Reply:
eb8ec10